### PR TITLE
dcos-3512 send CLUSTER_ID to segment.io

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

File: cli/dcoscli/analytics.py [hunk 1]
```diff
@@ -261,6 +261,12 @@ def _base_properties(conf=None):
         logger.exception('Unable to find the hostname of the cluster.')
         dcos_hostname = None
 
+    try:
+        cluster_id = mesos.DCOSClient().metadata().get('CLUSTER_ID')
```
Comment (jsancio) [inline cli/dcoscli/analytics.py]: @tamarrow Can you confirm that we only make this call when `core.reporting` is `True`. In other words we shouldn't make this call when reporting is disabled.

Comment (tamarrow) [inline cli/dcoscli/analytics.py]: yep, I added a test.

**Comment by tamarrow** [comment]
@jsancio bump

**Comment by jsancio** [comment]
LGTM! :ship: ing it!

**Pull request merged**